### DISCUSSION TITLE
Auto-erasing old videos from NFS server dump

### DISCUSSION THREAD

**Discussion by akivacp** [gtxaspec/wz_mini_hacks#498] [Q&A]
I have an NFS option, where several cameras are recording to my Ubuntu server. However, when the recording fills up, it doesn't really start erasing older footage from the folders, as I've set up in the app, because obviously the Wyze app doesn't know better.
I'm not very skilled in Linux, so I wouldn't be able to solve this on my own. What is the best script I can run (either from the camera side or the server side) that will automatically start erasing old videos from the NFS dump like the same way the Wyze app does?

**Comment by kohrar** [comment]
I used to have a cronjob that deletes files older than some number of days. Perhaps you can do something like this as a daily cronjob? This will delete all mp4 files older than 30 days.
```
find /recordings -name "*.mp4" -mtime +30 -exec rm {} \;
```

**Reply by akivacp** [reply]
Thanks. I think I did a similar script, but does it also empty the folders? I'm trying to copy the same way that Wyze does.

**Reply by corbolais** [reply]
The `find` command deletes (`rm`) the MP4 video files. When folders/directories are empty after the `find` command they remain in place.

To remove empty directories use this `find` command after the first one completed:

`find /recordings -type d -exec rmdir -p --ignore-fail-on-non-empty {} \;`

HTH

**Reply by akivacp** [reply]
thanks. ill try this
